lib/converter.js

```diff
@@ -70,9 +70,43 @@ function convert(url, options) {
             return;
         }
 
-        var result = "data:image/png;base64," + page.renderBase64("PNG");
-        system.stdout.write(result);
-        phantom.exit();
+        /*
+         * We want to wati until the standard deviation of the previous 100 FPS checks
+         * is less than 1.5, to ensure the svg has rendered properly. The render rate
+         * tends to fluctuate while rendering the svg, and level out when it is finished.
+        */
+        page.evaluate(function() {
+            var previousT = 0;
+            var t = 0;
+            var fpsList = [];
+
+            function stdev(values) {
+                var mean = values.reduce(function(a, b) { return a + b; }) / values.length;
+                var devSquared = values.map(function(a) { return Math.pow(a - mean, 2); }).reduce(function(a, b) { return a + b; });
+                return Math.sqrt(devSquared / (values.length - 1));
+            }
+
+            function calcFPS(now) {
+                t = now;
+                var fps = 1000 / (t - previousT);
+                fpsList.push(fps);
+                var std = stdev(fpsList.slice(fpsList.length - 100, fpsList.length));
+                if (std <= 1.5) {
+                    console.log("done");
```

Comment:
should we have some breakout method in case we never hit this? IE like if fps.length > x call it done

Reply:
How big does `fpsList` usually get? Might be a good idea to add kentons check.

Reply:
Yeah, but I think a timeout might be better, because the length of fpsList is dependent on the FPS, which is dependent on processor, so it'd be hard for me to choose a generic number.